packages/core/src/components/Dialog/Dialog.tsx

```diff
@@ -26,6 +26,10 @@ export interface DialogProps extends VibeComponentProps {
    * Class name applied to the reference wrapper element.
    */
   referenceWrapperClassName?: string;
+  /**
+   * The wrapper element type to use for React components. Defaults to "span".
+   */
+  referenceWrapperElement?: "span" | "div";
```

Comment:
I guess it's not - but just making sure - it isn't related to the PR right? sneaked in from some commit?